src/components/table/table.ts

```diff
@@ -50,6 +59,27 @@ export class MTable extends ModulVue {
         return this.rows.length === 0 && !this.loading;
     }
 
+    public sort(columnTable: MColumnTable): void {
+        if (columnTable.sortable) {
+            this.$emit('update:sortedColumn', {
+                dataProp: columnTable.dataProp,
```

Comment:
J'aurais p-t sorti ça dans un objet de type MSortedColumn et ensuite j'aurais passé cet objet dans l'emit. Ça permettrait de valider l'objet passé. Qu'en penses-tu?

Reply:
Je ne suis pas certain de comprendre quelle validation tu voudrais faire?

Reply:
Quand tu vas catcher le emit, tu vas t'attendre à ce qu'il soit de type MSortedColumn alors en lui passant réellement un objet de ce type-là, on se protégerais, je sais pas si tu me suis.. sinon c'est surement pas très important ;)

`const sortedColumn: MSortedColumn = new MSortedColumn();`
`sortedColumn.dataProp = xxxxxx;`
`sortedColumn.ascending = xxxxxx;`

`this.$emit('update:sortedColumn, sortedColumn)`

Reply:
Oui c'est bon. Je vais changer l'objet. C'est une interface actuellement, on ne pourra pas faire de new. Je vais en faire une class.

Reply:
C'est fait